### ISSUE TITLE
Is there any offline way to verify performance for discriminative setting?

### ISSUE THREAD

**Issue by JeremyLinky** [airbert-vln/airbert#9]
For  discriminative setting, do I need to upload the generated json file to (https://evalai.cloudcv.org/web/challenges/challenge-page/97/overview) every time to verify the model 's performance? If so, it seems very inconvenient, is there any faster or offline way?

**Comment by guhur** [comment]
This is a general rule: you SHOULD NOT test too many times on the test server. Otherwise, you might just be cheating, by overfitting on it. You should measure your performances on the validation set. And test on the server only when you are confident enough about your method.

**Comment by JeremyLinky** [comment]
> You should measure your performances on the validation set.

Thanks for your reply! However, I am still confused that how can I measure my performances on the validation set? Like I can directly obtain the SR and SPL?
